service/storage_service.cc

```diff
@@ -3827,7 +3830,7 @@ future<> storage_service::handle_state_normal(inet_address endpoint, gms::permit
         }
         auto current_owner = current->second;
         if (endpoint == current_owner) {
-            slogger.debug("handle_state_normal: endpoint={} == current_owner={} token {}", endpoint, current_owner, t);
+            slogger.info("handle_state_normal: endpoint={} == current_owner={} token {}", endpoint, current_owner, t);
```

Comment:
per-token printouts, and particularly for the unchanged case are a bit excessive.
why do we need them?

Reply:
I want to be able to distinguish the 4 branches.
Currently 2 branches contain an INFO log (which is already per-token) and 2 branches don't, so we don't have a way to distinguish which of the latter was taken.
By adding an INFO log to one of the two branches we're now distinguishing them all.

This branch is fortunately rare. The first branch is the most common one so it makes sense not to add a log message there.

Reply:
if it's rare then ok